tools/ledger/src/test/java/org/apache/bookkeeper/tools/cli/commands/cookie/GetCookieCommandTest.java

```diff
@@ -99,6 +99,7 @@ public void testGetCookieFromExistentCookieFile() throws Exception {
             .setInstanceId("test-instance-id")
             .setJournalDirs("/path/to/journal/dir")
             .setLedgerDirs("/path/to/ledger/dirs")
+                .setIndexDirs("/path/to/index/dirs")
```

Comment:
nit: it looks like we have some leading spaces

Reply:
Hmmm looks like a problem with my IntelliJ.. Good catch, will fix it.